src/components/PeopleTable.tsx

```diff
@@ -1,644 +1,191 @@
+import { useCallback, useContext, useMemo } from 'react';
+import { useParams, useSearchParams } from 'react-router-dom';
+
+import cn from 'classnames';
+import { StateContext } from '../store/GlobalContextProvider';
+import { Person } from '../types';
+import { SearchLink } from './SearchLink';
+import { compareValues } from '../utils/compareUtil';
+import { PersonLink } from './PersonLink';
+
 /* eslint-disable jsx-a11y/control-has-associated-label */
 export const PeopleTable = () => {
+  const { people } = useContext(StateContext);
+  const { slug: slugParam } = useParams();
+
+  const sortableHeaders = ['Name', 'Sex', 'Born', 'Died'];
+  const unsortableHeader = ['Mother', 'Father'];
+
+  const [searhParams] = useSearchParams();
```

Comment:
There is a typo in the variable name `searhParams`. It should be corrected to `searchParams` to match the intended meaning and improve code readability.